ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.16__amd64-rollback-nightly.yaml

```diff
@@ -87,6 +99,17 @@ tests:
           memory: 200Mi
     - ref: cucushift-chainupgrade-toimage
     workflow: cucushift-installer-rehearse-aws-ipi-byo-route53
+- as: aws-ipi-ovn-hypershift-inplace-rollback-f7
```

Comment:
The file name already contains `rollback`, so perhaps there is no need to add rollback to the test name, to avoid duplicate ? The job name is `periodic-ci-openshift-openshift-tests-private-release-4.16-amd64-rollback-nightly-aws-ipi-ovn-hypershift-inplace-rollback-f7`

Reply:
Done, Thanks